src/utils.cpp

```diff
@@ -766,5 +766,43 @@ bool executePreAction(const nlohmann::json& i_parsedConfigJson,
 
     return true;
 }
+
+std::string getFruPathFromJson(const nlohmann::json i_sysCfgJsonObj,
```

Comment:
const nlohmann::json& ?

Reply:
Why is this marked as resolved when the comment is still pending?

Reply:
where it is marked as resolved?

Reply:
I unresolved it just now.

Reply:
done